#goal definition
sensor_msgs/PointCloud2 tgt_cloud
sensor_msgs/JointState arm_home
---
#result definition
bool success
---
#feedback
bool object_moved
